<launch>

  <arg name="image"/>
  <arg name="camera_info"/>

  <node name="$(anon marker_detector)" pkg="pattern_pose_estimation" type="marker_detector_node" respawn="false" output="screen">
    <remap from="image" to="$(arg image)" />
    <remap from="camera_info" to="$(arg camera_info)" />
	<param name="marker_pattern" type="string" value="$(find ar_pose)/data/4x4/4x4_95.patt"/>
	<param name="marker_width" type="double" value="0.1"/>
	<param name="marker_center_x" type="double" value="0.0"/>
	<param name="marker_center_y" type="double" value="0.0"/>
	<param name="threshold" type="int" value="100"/>
	<param name="use_history" type="bool" value="true"/>
  </node>

</launch>

